### PR TITLE
new file

### PR DESCRIPTION
**Pull request by yann-lucas**
Yann PR

### CODE REVIEW

**Comment by yann-lucas** [comment]
no comment

File: Yann_file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a new file
```
Comment (PabloRdrRbl) [inline Yann_file.txt:1]: I have seen this file already, you sure it's new?

**Review by PabloRdrRbl — COMMENTED** [review]
Well done! I hope you keep using git!

File: Yann_file.txt (hunk 1 above)
Comment (yann-lucas) [reply Yann_file.txt:1]: Yes keep looking

**Review by PabloRdrRbl — APPROVED** [review]
It's ok! just fine

**Pull request merged**